src/core/AudioEngine/AudioEngineTests.cpp

```diff
@@ -3057,8 +3057,9 @@ void AudioEngineTests::throwException( const QString& sMsg ) {
 
 	pAE->setState( AudioEngine::State::Ready );
 	pAE->unlock();
-	
-	throw std::runtime_error( sMsg.toLocal8Bit().data() );
```

Comment:
Did asan complain about this? I'd have thought that runtime_error should take a copy of the error string, which is still in-scope here